earthpy/clip.py

```diff
@@ -155,5 +168,9 @@ def clip_shp(shp, clip_obj):
 
     if shp["geometry"].iloc[0].type == "Point":
         return _clip_points(shp, clip_obj)
+>>>>>>> ac2e97f37864baa8ddef53bfdb2f26eb318e43c5
```

Comment:
@nkorinek  will you please clean up this PR? there are merge conflicts still in it!